### PR TITLE
Contributors list on tag page sidebar

### PR DESCRIPTION
**Pull request by jimrandomh**
Puts a contributors list under the table of contents on tags. Based off https://github.com/LessWrong2/Lesswrong2/pull/4055 . Not ready for review, but if you wanted to experiment with the styling this might be useful. Doesn't have the hover-to-highlight-words-by-this-user thing at all yet.

### CODE REVIEW

**Comment by jimrandomh** [comment]
Now has hover-for-attribution. It doesn't always work right, but it works enough to be useful and to see how the UI works. (It's sensitive to the entire revision-history, so it's not necessarily obvious what's breaking it; when it's broken, highlights will wind up misaligned, and fall in weird places that aren't word boundaries.)

It's not quite ready to ship because of the aforementioned buggy misalignment, and that it currently recomputes the entire set of attributions from scratch on every pageload, the performance of which is usually fine but will occasionally be very-not-fine.

File: packages/lesswrong/components/tagging/TagPage.tsx [hunk 1]
```diff
@@ -223,84 +191,46 @@ const TagPage = ({classes}: {
...
-              <SubscribeTo
-                document={tag}
-                className={classes.subscribeTo}
-                showIcon
-                hideLabelOnMobile
-                subscribeMessage="Subscribe"
-                unsubscribeMessage="Unsubscribe"
-                subscriptionType={subscriptionTypes.newTagPosts}
+    {hoveredContributorId && <style>
+      {`.by_${hoveredContributorId} {background: #8f8;}`}
```
Comment (Discordius) [inline packages/lesswrong/components/tagging/TagPage.tsx:195]: Consider doing this with data selectors instead of custom classNames

**Comment by Discordius** [comment]
Played around a bit with styling, this feels overall better to me. In particular note the increased padding from 6px to 8px above the Load More button: 

image

**Comment by jimrandomh** [comment]
Our diff library breaks the assumption that no characters are inserted/removed without a corresponding `<ins>` or `<del>` tag when the contents of the ins/del would be only whitespace characters. This breaks on eg [rationality tag revision 0.11](https://www.lesswrong.com/compare/tag/rationality?before=0.0.10&after=0.0.11) (which inserts a paragraph-break).

**Comment by jimrandomh** [comment]
(Added handling to make the ToC and attributions work when you specify a historical revision. I needed this to debug what in the history was scrambling the attributions on some tag pages, and it's useful in-general.)

**Comment by jimrandomh** [comment]
Denormalized, attribution-alignment problems fixed, everything should be working. I applied the vertical padding tweaks from your screenshot, but not the horizontal alignment change; I see what you were going for moving it up an indent level to make an even left edge, but unfortunately when the karma scores are nonzero and variable width, that left edge turns into a mess and it looks better not trying to be horizontally aligned with the heading and the Load More.

**Review by Discordius — APPROVED** [review]
Overall seems good. I definitely can't rule out any tricky bugs in the diffing section, but the overall architecture seems reasonable.

**Review by Discordius — APPROVED** [review]

**Pull request merged**